docs/hw/jetson_hookup.md

```diff
@@ -51,7 +51,7 @@ This can be powered from the unregulated battery port of the Create® 3 adapter
 The [Jetson Xavier NX™ Developer Kit](https://developer.nvidia.com/embedded/jetson-xavier-nx-devkit) has a 5.5 mm x 2.5 mm barrel connector jack (J16) for power (9 V to 20 V) and a USB Micro-B port (J5) for upstream data.<br>
 This can be powered from the unregulated battery port of the Create® 3 adapter board by using a JST-XH female connector to DC barrel plug cable.
 
-The data connection is established by using a USB Micro B to USB-C® cable, which requires some software tweaking to allow the Create3-to-Jetson communication. See [this page](../setup/jetson.md).
+The data connection is established by using a USB Micro-B to USB-C® cable, which requires some software tweaking to allow the Create3-to-Jetson communication. See [this page](../setup/jetson.md).
```

Comment:
I think it is actually supposed to be "USB Micro B" (without a hyphen, despite it having to be USB-C®)

Reply:
I now see there are a few other instances of this (including line 51)

Reply:
I'm fine with changing to "USB Micro B" all at once, as it seems that USB-IF back then was not standardizing the notation very well, so everybody has a different way.
But just to share the current state on USB-IF website, you tend to see Micro-B notation more.
https://www.google.com/search?q=%22micro+b%22+on%3Ausb.org

Reply:
[7fad7ac](https://github.com/iRobotEducation/create3_docs/pull/385/commits/7fad7ac549ddff55294865e8fba9f1c7c227bb63)